src/foam/nanos/approval/PopulateApprovalRequestSummariesAdapter.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * NANOPAY CONFIDENTIAL
+ *
+ * [2022] nanopay Corporation
+ * All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of nanopay Corporation.
+ * The intellectual and technical concepts contained
+ * herein are proprietary to nanopay Corporation
+ * and may be covered by Canadian and Foreign Patents, patents
+ * in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from nanopay Corporation.
+ */
+
+package foam.nanos.approval;
```

Comment:
Please add a services.jrl entry on the foam side that adds this adaption - that the nanopay services entry will override.